packages/gridtools/package.py

```diff
@@ -13,7 +13,7 @@ class Gridtools(CMakePackage):
     git = "git@github.com:GridTools/gridtools.git"
 
     maintainers = ['elsagermann']
-    
+
```

Comment:
looks like you only added remove spaces, you may revert to master if no on purpose

Reply:
I want to remove trailing spaces yes! Sorry for the trouble